### PR TITLE
Use Process 2017 and accept 2015 for now

### PR DESCRIPTION
**Pull request by plehegar**
Address #565 
https://lists.w3.org/Archives/Member/chairs/2017JanMar/0007.html

Do not deploy before 1 March 2017.

@marcoscaceres @tabatkins Documents that link to Process 2015 will be rejected starting on April 1 2017.

### CODE REVIEW

**Comment by deniak** [comment]
@plehegar I understand (from #565) people will still be able to use process 2015 until April 1, 2017. If so, I will implement the 1-month transition period in the rule.

**Comment by tripu** [comment]
@plehegar, missing update to tests about the process: [`travis-ci.org/w3c/specberus/jobs/192969083`](https://travis-ci.org/w3c/specberus/jobs/192969083).

**Comment by deniak** [comment]
@tripu yes, I told @plehegar I would look into the tests and also look if there's a better way to handle the transition period instead of having to create a new PR when it ends.

**Comment by deniak** [comment]
@plehegar @tripu I update the PR to implement the 1-month transition period. It can be merged and deployed today and no need to create a new PR after the transition period. Can I ask one of you to review the changes? Thanks!

File: lib/l10n-en_GB.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ exports.messages = {
 ,   "sotd.group-homepage.no-key":       `This instance of Pubrules is missing a valid <a href="https://w3c.github.io/w3c-api/#apikeys">key
         for the W3C API</a>; contact <a href="mailto:sysreq@w3.org"><code>sysreq@w3.org</code></a> for help.`
     // sotd/process-document
-,   "sotd.process-document.deprecated":        "It seems the document attemps to follow process 1 Aug 2014, which is deprecated in favour of <a href='https://www.w3.org/2015/Process-20150901/'>1 September 2015</a>."
+,   "sotd.process-document.deprecated":        "The process document you are using is not longer allowed."
```
Comment (tripu) [inline lib/l10n-en_GB.js]: Should be &ldquo;no longer&rdquo; (no *t*).

File: lib/rules/sotd/process-document.js [hunk 2]
```diff
@@ -25,7 +39,11 @@ exports.check = function (sr, done) {
                 }
             }
             else if (deprecatedBoilerplate === sr.norm($p.text()) || deprecatedUri === $p.find('a').attr('href')) {
-                sr.error(self, 'deprecated');
+                if (deprecatedAllowed) {
+                    sr.warning(self, 'deprecated');
+                    found = true;
+                }
+                else sr.error(self, 'deprecated');
```
Comment (tripu) [inline lib/rules/sotd/process-document.js]: Do you think it'd be helpful to pass along the detected process as a parameter, to be displayed as part of the message?

**Comment by deniak** [comment]
Oups, indeed, I forgot to commit the new files.

File: lib/l10n-en_GB.js (hunk 1 above)
Comment (deniak) [inline lib/l10n-en_GB.js]: addressed

**Pull request merged**